config/dato/menu-helpers.mjs

```diff
@@ -5,12 +5,22 @@ import mkdirp from 'mkdirp'
 
 function buildChildrenTree(items) {
   items.forEach(item => {
-    
     if (item.children) {
-      item.children.forEach(child => {
+      item.children = item.children.map(child => {
```

Comment:
It's nitpicking really, but I think this part can be improved by making it more clear what it is doing. I would rename `child` to `updatedChild`. You might be able to destructure `child` so you don't have to write it multiple times.

It can look something like this:
```
foo = foo.map(({ layerInstance, tags, ...childNode }) => {
  let updatedChildNode = { ...childNode }

  if (layerInstance) {
   [...]
  }

  if (tags) {
    [...]
  }

  return updatedChildNode
});
```